### PR TITLE
New Moves Added

### PR DESCRIPTION
**Pull request by Delta456**
Crunch
Will-O-Wisp
Extreme Speed
Ancient Power

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -9,3 +9,9 @@ test.sh
 output.txt
 moves completed.txt
 move_cb_timings.txt
+deps/armips
+deps/armips
+libiconv2.dll
+libintl3.dll
+msvcp140.dll
+deps/armips
```
Comment (EternalCode) [inline .gitignore]: I don't understand why ARMIPs was added to your gitignore along with some C++ runtime DLLs. The DLLs aren't even supposed to be a part of the project. Please revert changes to this file.

File: src/moves/basic_move_procs.c [hunk 2]
```diff
@@ -1072,3 +1072,39 @@ struct move_procs target_lower_def_50_procs = {
...
+struct move_procs burn_100_procs = {
+    // Atk, Def, Spd, SpA, SpD, Evn, Acc, Crt
+    {0, 0, 0, 0, 0, 0, 0, 0},
+    {0, 0, 0, 0, 0, 0, 0, 0},
+    {0, 0, 0, 0, 0, 0, 0, 0},
+    {0, 0, 0, 0, 0, 0, 0, 0},
+    AILMENT_NONE,
+    AILMENT_BURN,
+    0,
+	100,
```
Comment (EternalCode) [inline src/moves/basic_move_procs.c]: Consistent tabbing, and end a file with a new line please.

File: src/test_data.c [hunk 3]
```diff
@@ -4,20 +4,20 @@
 #include "battle_data/battle_state.h"
 
 /* Player data */
-const static u16 player_species = SPECIES_GARDEVOIR;
-const static u16 player_move_1 = MOVE_GRASSY_TERRAIN;
-const static u16 player_move_2 = MOVE_PSYCHIC_TERRAIN;
-const static u16 player_move_3 = MOVE_ELECTRIC_TERRAIN;
-const static u16 player_move_4 = MOVE_MISTY_TERRAIN;
-const static u8 player_level = 60;
+const static u16 player_species = SPECIES_BLASTOISE;
```
Comment (EternalCode) [inline src/test_data.c]: Changes to the test file shouldn't be committed and pushed. Don't add it to your git ignore, just don't commit it either. Similar to the dependencies when on Windows.

File: .gitignore (hunk 1 above)
Comment (EternalCode) [inline .gitignore]: I don't understand why ARMIPs was added to your gitignore along with some C++ runtime DLLs. The DLLs aren't even supposed to be a part of the project. Please revert changes to this file.

File: src/moves/basic_move_procs.c [hunk 4]
```diff
@@ -1072,3 +1072,39 @@ struct move_procs target_lower_def_50_procs = {
...
+};
```
Comment (EternalCode) [inline src/moves/basic_move_procs.c]: Consistent tabbing, and end a file with a new line please.

File: src/test_data.c [hunk 5]
```diff
@@ -4,20 +4,20 @@
...
 
 /* Opponent data */
-const static u16 opponent_species = SPECIES_CHARMANDER;
+const static u16 opponent_species = SPECIES_BULBASAUR;
 const static u16 opponent_move_1 = MOVE_NATURE_POWER;
 const static u16 opponent_move_2 = MOVE_NATURE_POWER;
 const static u16 opponent_move_3 = MOVE_NATURE_POWER;
 const static u16 opponent_move_4 = MOVE_NATURE_POWER;
-const static u8 opponent_level = 60;
+const static u8 opponent_level = 50;
```
Comment (EternalCode) [inline src/test_data.c]: Changes to the test file shouldn't be committed and pushed. Don't add it to your git ignore, just don't commit it either. Similar to the dependencies when on Windows.

Comment (Delta456) [reply src/test_data.c]: Ok

File: src/moves/basic_move_procs.c (hunk 4 above)
Comment (Delta456) [reply src/moves/basic_move_procs.c]: Fixed

File: .gitignore (hunk 1 above)
Comment (Delta456) [reply .gitignore]: Done

File: src/moves/basic_move_procs.c (hunk 4 above)
Comment (EternalCode) [reply src/moves/basic_move_procs.c]: Line 1112.

File: src/test_data.c [hunk 6]
```diff
@@ -4,12 +4,21 @@
...
 const static u16 player_move_4 = MOVE_MISTY_TERRAIN;
 const static u8 player_level = 60;
+=======
+-const static u16 player_species = SPECIES_GARDEVOIR;
+-const static u16 player_move_1 = MOVE_GRASSY_TERRAIN;
+ -const static u16 player_move_2 = MOVE_PSYCHIC_TERRAIN;
+ -const static u16 player_move_3 = MOVE_ELECTRIC_TERRAIN;
+ -const static u16 player_move_4 = MOVE_MISTY_TERRAIN;
+ -const static u8 player_level = 60;
+>>>>>>> 5aa34cd877f9cf8bf4e31abf94ef9474d86af0fa
```
Comment (EternalCode) [inline src/test_data.c]: Unresolved merge conflict.

**Pull request merged**